File: app/src/main/java/za/co/woolworths/financial/services/android/models/dto/LiquorCompliance.java
package za.co.woolworths.financial.services.android.models.dto;

import android.os.Parcelable;

public class LiquorCompliance  {
    private String liquorImageUrl;
    private boolean liquorOrder=false;


    public LiquorCompliance(boolean liquorOrder,String liquorImageUrl){
        this.liquorOrder=liquorOrder;
        this.liquorImageUrl=liquorImageUrl;


    }

    public boolean isLiquorOrder() {
        return liquorOrder;
    }

    public void setLiquorOrder(boolean liquorOrder) {
        this.liquorOrder = liquorOrder;
    }



    public String getLiquorImageUrl() {
        return liquorImageUrl;
    }

    public void setLiquorImageUrl(String liquorImageUrl) {
        this.liquorImageUrl = liquorImageUrl;
    }


}

File: app/src/main/java/za/co/woolworths/financial/services/android/models/dto/LiquorCompliance.kt
package za.co.woolworths.financial.services.android.models.dto

class LiquorCompliance(liquorOrder: Boolean, liquorImageUrl: String) {
    var liquorImageUrl: String
    var isLiquorOrder = false

    init {
        isLiquorOrder = liquorOrder
        this.liquorImageUrl = liquorImageUrl
    }
}